platform/spire/src/virt.py

```diff
@@ -288,12 +288,14 @@ def generate_qemu_args(self):
         args = ["qemu-system-x86_64"]
         args += ["-nographic", "-serial", "mon:stdio"]
         args += ["-machine", "accel=kvm", "-cpu", "host"]
-        args += ["-hda", self.hd]
         if self.cd is None:
+            args += ["-drive", "media=disk,file=" + self.hd]
             args += ["-boot", "c"]
         else:
-            args += ["-cdrom", self.cd]
-            args += ["-boot", "d"]
+            # TODO: maybe set this readonly
+            args += ["-drive", "media=disk,format=raw,file=" + self.cd]
+            args += ["-drive", "media=disk,file=" + self.hd]
+            args += ["-boot", "c"]
```

Comment:
Could we leave it configurable whether to put the installer as CD-ROM or hard disk?

Reply:
We could, but do we think that's useful?

Reply:
I don't see it as out of the question that someone might want to use an actual CD-ROM for install media.